pkg/pattern/aggregation/push.go

```diff
@@ -202,6 +213,13 @@ func (p *Push) buildPayload() ([]byte, error) {
 
 	payload = snappy.Encode(nil, payload)
 
+	sp.LogKV(
+		"event", "build aggregated metrics payload",
+		"services", strings.Join(services, ","),
```

Comment:
Should the "services" be limited somehow? I'm not expecting a lot but just in case there is a large number of services we're trying to attach to a span.